Day 01 - GIt Pull and Merge/Task01 - Git Push and Merge

```diff
@@ -0,0 +1 @@
+This is the first Push from remote system
```

Comment:
could you please, add a newline at the end of file.

Reply:
@geoffrey1330  updated as suggested